src/snyk/base/services/authenticationService.ts

```diff
@@ -106,6 +113,16 @@ export class AuthenticationService implements IAuthenticationService {
 
     if (!token) return;
     await this.configuration.setToken(token);
+    // TODO remove feature flag when ready
+    if (!this.configuration.getPreviewFeatures().lsAuthenticate) {
+      return Promise.resolve();
+    }
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
You don't need this. Also we should never assign unsafe args, unless there's no really way around it.